src/themes/future/modules/banners/stats.tpl

```diff
@@ -0,0 +1,37 @@
+{if $smarty.const.NV_IS_BANNER_CLIENT}
+<ul class="nav nav-tabs m-bottom">
```

Comment:
@hainguyen0990 giao diện mới không có các class `m-bottom` này nữa. Em cần rà soát và viết lại toàn diện, không đơn thuần là sửa cái cũ sang smarty

Reply:
Ngoài ra viết như này sai cấu trúc HTML của nav-tabs bootstrap 5. Xem ở đây https://getbootstrap.com/docs/5.3/components/navs-tabs/